public/assets/sass/admin/generic-editor.scss

```diff
@@ -96,3 +96,39 @@
     }
   }
 }
+
+.modal {
+  display: none;
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
```

Comment:
This `z-index` doesn't do anything for us, since we don't really use z-indexes around the codebase.

Reply:
removing the z-index puts the modal behind the footer on the editor page, so i'm gonna leave it there